### ISSUE TITLE
OpenSSL 1.1.0

### ISSUE THREAD

**Issue by akire** [Microsoft/openssl#38]
Are there any plans on integrating the OpenSSL 1.1.0 release into this fork?

**Comment by khouzam** [comment]
Yes, this is something we want to do. There has been a lot of changes between 1.0.x and 1.1.0. Including a lot of build changes. I'm hoping to work with the OpenSSL team and do the work so that we can get this move up to the main OpenSSL repo and eventually fully supported.

@richsalz would you be the right person to talk to?

**Comment by richsalz** [comment]
Start by posting on the openssl-dev mailing list.  Then someone from the team can help with specifics, if needed.

**Comment by akire** [comment]
Any update on this @khouzam ?

**Comment by khouzam** [comment]
Unfortunately, I haven't had a chance to move forward on this yet. I should find some time soon to start looking.

**Comment by supercairos** [comment]
Hi, any ETA on that ? :)

**Comment by khouzam** [comment]
Unfortunately not. I haven't had any time to start looking at this issue.

**Comment by supercairos** [comment]
No problem, I backported my code to OpenSSL 1.0.2. Was easier than expected :)
